parsec/backend/memory/user.py

```diff
@@ -253,26 +253,16 @@ async def find_humans(
     ) -> Tuple[List[HumanFindResultItem], int]:
         org = self._organizations[organization_id]
```

Comment:
`query` typing is wrong, should be `Optional[str]`

Reply:
but why removing the default value ?

Reply:
fixed